### DISCUSSION TITLE
Test Corrección T3

### DISCUSSION THREAD

**Discussion by smrebolledo** [IIC2133-PUC/2024-1#199] [Tarea 3]
Hola, podrían subir los test que se usaron para corregir la parte 3 porfis, quiero recorregir pero con los tests iniciales que daban a mi si me funcionaba entonces necesito saber como son esos para saber que tengo mal.

O para recorregir en esta parte está considerado con los test cases iniciales?
![image](https://github.com/IIC2133-PUC/2024-1/assets/42575575/434d689f-7a55-4c64-895c-f7581a346f33)

**Comment by benjavicente** [comment]
Hola! Son los mismos entregados públicos por lo que revise.

No funcionó el mensaje de **error de memoria**, parece ser eso, voy a tratar arreglar eso y actualizar los errores lo antes posible.

**Reply by benjavicente** [reply]
```sh
valgrind --tool=massif --massif-out-file=massif_output.txt [programa y argumentos]
ms_print massif_output.txt | head -50
```
